### ISSUE TITLE
Clojure-lsp not getting past initialization phase after update of vscode

### ISSUE THREAD

**Issue by bpringe** [BetterThanTomorrow/calva#1056]
This problem was reported as happening on Windows, not sure if it happens elsewhere. As reported on Slack, these steps resolved the issue:

```
rm -rf .lsp/sqlite.db
rm -rf .clj-kondo/.cache
mv .clj-kondo/config.edn .clj-kondo/config.backup.edn
```

**Comment by bpringe** [comment]
I haven't seen this issue myself after upgrading, on both Linux and Windows. Will leave open for a bit longer to see if we get more reports.

**Comment by Limess** [comment]
I just experienced what I think was the same issue on windows after an update, the steps above resolved the issue for me, although my issue seemed to be related to jack-in rather than lsp initiation so it could be separate?


For reference the last thing I saw after jacking in (which showed as successful) was `; Happy coding! ♥️` in the output window, there was no `; Jacking in...` output in that window.

I _think_ I've seen the same thing on my work Mac but didn't dig into it at the time, however I have seen this behaviour before where the jack-in doesn't complete, and thinking back it was probably after a VSCode update.
